source/dotnet/Contracts/ProcessCompleted.cs

```diff
@@ -21,4 +21,9 @@ public partial class ProcessCompleted
     /// The message type for balance-fixing-completed transport message meta data in accordance with ADR-008.
     /// </summary>
     public const string BalanceFixingProcessType = "BalanceFixingCompleted";
+
+    /// <summary>
+    /// The message type for aggregation-completed transport message meta data in accordance with ADR-008.
+    /// </summary>
+    public const string AggregationProcessType = "AggregationProcessType";
```

Comment:
I'd probably stick with just
```suggestion
    public const string AggregationProcessType = "Aggregation";
```
I'm however wondering with the balance fixing string is named like an event...

Reply:
Name changed.

Reply:
Or perhaps? (and similar for balance fixing)
```suggestion
    public const string AggregationCompletedEventName = "AggregationCompleted";
```

Reply:
Next time.